workloads/imagenet/imagenet_jax/input_pipeline.py

```diff
@@ -0,0 +1,262 @@
+# Copyright 2021 The Flax Authors.
```

Comment:
here and in `models.py`, remove this copyright and instead add a comment saying it was forked from the flax repo with a link to the file it was forked from. IANAL but my understanding is this is sufficient under the Apache 2.0 license, because this repo is also Apache 2.0